ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js

```diff
@@ -1,3 +1,4 @@
+var sku = "24-WG087";
```

Comment:
@ashishgoyalweb : why is this hard-coded. Even if we this is an assignation it should be ""

Reply:
As per the discussion with Rajamouli, The sku parameter will be coming from PLP page or home page when the user click on a product from that page.  Right now we don't have those pages that is why he hardcoded the sku value.